src/foam/u2/wizard/wao/SplitWAO.js

```diff
@@ -48,10 +48,10 @@ foam.CLASS({
     },
     async function load (wizardlet) {
       if ( wizardlet.loading ) return;
+      wizardlet.loading = true;
```

Comment:
I think loading before super call was breaking something. @KernelDeimos do you recall?

Reply:
I would expect this to break things. This will prevent the delegate WAO from actually running (supposing it checks that `wizardlet.loading` property)